ITE/4/README.adoc

```diff
@@ -233,22 +232,26 @@ properties. For example, an implementation that is capable of hashing GitHub
 entities may choose to use a subsection of the contents obtained using the
 official API, which returns JSON representations containing different properties
 of the entities. It is important for these representations to be consistent for
-a particular entity and they must be generated keeping in mind the different
-sources of non-determinism as highlighted in the Appendix. It's also necessary
-for this method to handle missing resources safely.
-
-Being able to get some hashable representation of abstract entities is necessary
-for in-toto inspections to verify contents as the context demands it. Therefore,
-it should be possible for implementations to use the method in scenarios other
-than just the recording of hashes of an artifact.
-
-It is quite likely that a generic URI specified may point to a collection of
+a particular entity, and to acknowledge the different
+sources of non-determinism highlighted in the Appendix. It is also necessary
+for this method to handle any missing resources safely. That is, if a resource does not exist
+at the location pointed to by a URI, this method must fail in a secure manner.
+
+Being able to get some hashable representation of abstract entities is also needed
+for in-toto inspections to verify contents. There is no specification mandating
```

Comment:
I apologise, I think this is on me, but what I meant was that the in-toto specification doesn't mandate any particular inspections, but leaves it open as a way for supply chain owners to verify anything they need to over and above the regular verification workflow. Therefore, it'd be necessary for them resolve abstract entities into some consistent form that `get_hashable_representation` delivers.

Reply:
I'm not sure if that meaning quite changed, although I may  be missing a crucial part of either sentence.